mscore/events.cpp

```diff
@@ -729,8 +729,14 @@ void ScoreView::mouseDoubleClickEvent(QMouseEvent* mouseEvent)
 
       Element* clickedElement = elementNear(toLogical(mouseEvent->pos()));
 
-      if (!(clickedElement && clickedElement->isEditable()))
+      if (!clickedElement)
             return;
```

Comment:
Please enclose this in braces.

Reply:
`Because the `else` branch needs them, the `if` branch should have them too